modules/genomic_browser/test/TestPlan.md

```diff
@@ -286,22 +283,17 @@ The following columns should be presented
 - The file content should follow the filtered values of the tab.
 
 ### View Genomic File
-- The view genomic file page should provide a *Download* button for each file.
+- The "Files" tab (or Genomic File Uploader subpage) should provide a *Download* button for each file.
 - Clicking on the *Download* button should trigger a file download.
 
 <a name="data_upload_link">
 </a>
 
 ## Data Upload
 
-> Under construction
-
-<a name="help_link">
-</a>
-
-## Help
-
-***
+Files may be uploaded under the Files tab (Genomic File Uploader subpage) IFF the Genomic Browser back-end is properly customized/configured and sample files of the appropriate format are available for upload testing.
```

Comment:
`IFF`

Reply:
Stands for "if and only if". I think it's probably better to spell it out rather than rely on this abbreviation. It's come up elsewhere that people don't know what it means.

Reply:
Ahh, I was confused by this last week as well. I'm not sure if "IFF" is popular and I just never have encountered it in my life up until now. Maybe it's alright? @christinerogers

Reply:
@maltheism I've come across it in academic contexts but not since undergrad... I added an issue to change this in our docs #6420